packages/resolve-scripts/src/template/client/store/index.js

```diff
@@ -1,11 +1,17 @@
 import axios from 'axios';
 import { createStore, applyMiddleware } from 'redux';
-import { sendCommandMiddleware } from 'resolve-redux';
+import {
+  sendCommandMiddleware,
+  setSubscriptionMiddleware
+} from 'resolve-redux';
 import reducer from '../reducers';
 
 const middleware = [
     sendCommandMiddleware({
         sendCommand: async command => axios.post(`${process.env.ROOT_DIR}/api/commands`, command)
+    }),
+    setSubscriptionMiddleware({
+        rootDirPath: process.env.ROOT_DIR
```

Comment:
Maybe it will be better to pass there socket instead of only rootDirPath. Because in this case user can add custom subscription  if it needed for him.